src/App.vue

```diff
@@ -75,6 +75,15 @@ export default {
     }
   },
 
+  // autoconnect on start
+  mounted: function() {
```

Comment:
Je zet hier de devmode altijd op true, is dat de bedoeling? Kun je niet de parameter defaulten naar true?